packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/buttonHeight.st

```diff
@@ -1,4 +1,4 @@
-accessing
+building
```

Comment:
Sure this is primarily used within building but considering that it returns a constant it should probably be `accessing`. What do you think?

```suggestion
accessing
```